lib/server.js

```diff
@@ -302,6 +302,11 @@ class Server extends KarmaEventEmitter {
         }
         executor.schedule()
       })
+
+      this.on('stop', function (done) {
+        this.log.debug('Exiting.')
```

Comment:
How about 'Received stop event, exiting' or something that makes it clear why we exit.

Reply:
Sounds good, will change.